js-practive/js/main.js

```diff
@@ -1 +1,260 @@
-// Place your code here, or you can split it how you what. It's up to you 😎
\ No newline at end of file
+// console.log(event)
+    // 1.очищення всієї сторінки 
+    // 2.умови, що знаходяться в хещі та їх валідація (асинхронний запит який вибудовує сторінку) (вбудувати сторінку )
+let usersTable
+let createButton
+let userForm 
+
+setUsersTable()
+setHtmlElementUserForm()
+
+function setUsersTable() {
+    usersTable = document.getElementById('users-table-list');
+    createButton = document.getElementById('create');
+
+    createButton.addEventListener('click', goToUserForm);
+    usersTable.addEventListener('click', onUsersTableClick);
+}
+
+function setHtmlElementUserForm() {
+    userForm = document.getElementById('users-edit');
+    userForm.addEventListener('click', onUserFormClick);
+}
+
+readingHashState();
```

Comment:
why do you add it in scope of function. 
Moreover function is only calling one function inside `addEventListener  `..

In nutshel function `readingHashState  ` is useless.